src/reducers/versions.tsx

```diff
@@ -756,7 +759,13 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
     case getType(actions.loadDiff): {
       const { baseVersionId, headVersionId, version } = action.payload;
 
-      const { entries, selectedPath } = getVersionInfo(state, headVersionId);
+      const headVersion = getVersionInfo(state, headVersionId);
+      if (!headVersion) {
+        _log.debug(`Version missing for headVersionId: ${headVersionId}`);
```

Comment:
Should this be `log.warn` because this seems like a pretty critical situation? Just returning `state` makes sense, but it will appear as if nothing happened. I suppose `debug` will also put a message in the console, but `warn` seems more appropriate.

Reply:
I don't really when this is supposed to occur to be honest.